### PR TITLE
Wolves and watchers can only smash mineable walls and not normal walls.

### PR DESCRIPTION
**Pull request by PetMudstone**
## About The Pull Request
Title. Wolves and watchers can only destroy mineable walls now. This means they can no longer destroy artificial walls.

## Why It's Good For The Game
It was dumb that weaker mobs such as wolves and watchers were capable of destroying ship walls. This PR prevents them from doing that while still allowing them the ability to break mineable walls.

## Changelog
:cl:
balance: Wolves and watchers are now incapable of destroying artificial walls and may only destroy mineable walls.
/:cl:

### CODE REVIEW

File: code/__DEFINES/mobs.dm [hunk 1]
```diff
@@ -242,8 +242,9 @@
 //determines if a mob can smash through it
 #define ENVIRONMENT_SMASH_NONE			0
 #define ENVIRONMENT_SMASH_STRUCTURES	(1<<0) 	//crates, lockers, ect
-#define ENVIRONMENT_SMASH_WALLS			(1<<1)  //walls
-#define ENVIRONMENT_SMASH_RWALLS		(1<<2)	//rwalls
+#define ENVIRONMENT_SMASH_MINERALS		(1<<1)	//minable walls
+#define ENVIRONMENT_SMASH_WALLS			(1<<2)  //walls
+#define ENVIRONMENT_SMASH_RWALLS		(1<<3)	//rwalls
```
Comment (ZephyrTFA) [inline code/__DEFINES/mobs.dm]:
```suggestion
#define ENVIRONMENT_SMASH_MINERALS (1<<1) //minable walls
#define ENVIRONMENT_SMASH_WALLS (1<<2) //walls
#define ENVIRONMENT_SMASH_RWALLS (1<<3) //rwalls
```

**Comment by shiptest-server** [comment]
#### Test Merge Removed

**Comment by shiptest-server** [comment]
#### Test Merge Deployed

##### Server Instance
shiptest

##### Revision
Origin: aabc6f1b144d0c36cf9f9f5d49313c6586ec6706
Pull Request: 15eea4e8892a081987ac1c9f6ad3a2b25f851e43
Server: 4a612b6abc77d3aaae270b65dc4c94667e4d6be2

**Comment by shiptest-server** [comment]
#### Test Merge Updated

##### Server Instance
shiptest

##### Revision
Origin: 293bad3e54aa3f26644d5d871baee9eda36057ca
Pull Request: 15eea4e8892a081987ac1c9f6ad3a2b25f851e43
Server: 2c17e761a3dbe846412b0eb5224178d8c7dfc1e4

**Comment by shiptest-server** [comment]
#### Test Merge Updated

##### Server Instance
shiptest

##### Revision
Origin: d79659920aec58007df63d3e9b96cb0fe1a16353
Pull Request: 15eea4e8892a081987ac1c9f6ad3a2b25f851e43
Server: 72d94bef2620899bd315f660ccb547df14ffc328

**Comment by shiptest-server** [comment]
#### Test Merge Updated

##### Server Instance
shiptest

##### Revision
Origin: dd854e7386a7a9a664d6357aab6213b032e1817b
Pull Request: 15eea4e8892a081987ac1c9f6ad3a2b25f851e43
Server: d097fa6e32a13458e756307a43c1764f99b243bc

**Comment by shiptest-server** [comment]
#### Test Merge Updated

##### Server Instance
shiptest

##### Revision
Origin: 4d98cdcea7b4591b94a06237309039f4980217fa
Pull Request: 15eea4e8892a081987ac1c9f6ad3a2b25f851e43
Server: 9a29cfa82da92d64f18a8f8bcd69d3ea934be1ac

**Comment by shiptest-server** [comment]
#### Test Merge Updated

##### Server Instance
shiptest

##### Revision
Origin: f789e74004b3acb82690d77398848b240a153b6f
Pull Request: 15eea4e8892a081987ac1c9f6ad3a2b25f851e43
Server: fec71ade6e716fb63d267abca95b86e22f5eb182

**Comment by shiptest-server** [comment]
#### Test Merge Removed

**Comment by shiptest-server** [comment]
#### Test Merge Deployed

##### Server Instance
shiptest

##### Revision
Origin: 033ea7d0a6e5ade2feced249d03f7bfcd3eac439
Pull Request: 15eea4e8892a081987ac1c9f6ad3a2b25f851e43
Server: 7c123be6a9d9ffcf192aa21466691c922a728b54

**Comment by shiptest-server** [comment]
#### Test Merge Updated

##### Server Instance
shiptest

##### Revision
Origin: 9693b436ac8eb8d68349b284df72bb252209581f
Pull Request: 15eea4e8892a081987ac1c9f6ad3a2b25f851e43
Server: 651bb7bd5250da841cf57899f6ff939ae8b47202

**Comment by shiptest-server** [comment]
#### Test Merge Removed

**Pull request merged**